stages/org.osbuild.users

```diff
@@ -94,6 +94,8 @@ def useradd(root, name, uid=None, gid=None, groups=None, description=None, home=
     if home:
         arguments += ["--home-dir", home]
         arguments += ["--create-home"]
+    elif home == "":
+        arguments += ["--no-create-home"]
```

Comment:
Why check for an empty home rather than passing `--no-create-home` unconditionally if it is not set?

Reply:
If `home` is not specified at all, you get the default behaviour, which is to create the default home dir for the user. See, e.g. [`test`](https://github.com/osbuild/osbuild/blob/5112f72cbf0caea7699b5ac77a60c903c9c57561/test/data/stages/users/b.mpp.json#L36) has no `home` option and gets [`home/test`](https://github.com/osbuild/osbuild/blob/5112f72cbf0caea7699b5ac77a60c903c9c57561/test/data/stages/users/diff.json#L10). If `home` is specified, it gets that home dir (not tested actually) if `home` is specified but empty, you get no home dir (this patch).